code/src/pages/NotFound.js

```diff
@@ -0,0 +1,11 @@
+/* eslint-disable react/no-unescaped-entities */
+import React from 'react'
+
+export const NotFound = () => {
+  return (
+    <div id="not-found-header"> WRONG WAY, <br /> GO BACK!
+      <p id="not-found-smaller">This page doesn't exist.</p>
+      <p id="not-found"><a href="/" tabIndex="0">Take me back home.</a></p>
+    </div>
+  )
+}
```

Comment:
I could only get your custom page not found on netlify.
To make this work on deployed version on netlify create file `_redirects `in public folder with the following code inside it  `/* /index.html 200`

Reply:
added! thank you!